lib/core/bytes.nit

```diff
@@ -613,6 +651,58 @@ private class BytesIterator
 	redef fun item do return tgt[index]
 end
 
+redef class Int
+	# Returns the big-endian representation of self.
```

Comment:
I don't understand why is this `big-endian` without parenthesis when the previous comment states:

> Returns self interpreted as a (big-endian) positive integer.

Is the endianess more important in the second method? Am  I missing something?